cli/cmd/upgrade.go

```diff
@@ -67,7 +68,10 @@ func (options *upgradeOptions) upgradeOnlyFlagSet() *pflag.FlagSet {
 		&options.force, "force", options.force,
 		"Force upgrade operation even when issuer certificate does not work with the trust anchors of all proxies",
 	)
-
+	flags.BoolVar(
+		&options.addOnOverwrite, "addon-overwrite", options.addOnOverwrite,
+		"Skip add-on config from linkerd-config-addons, and overwrite with the default or passed addon-config",
```

Comment:
Given people might not be aware of the existence of `linkerd-config-addons`, could you rewrite this to something like "Overwrite (instead of merge) existing add-ons config with file in --addon-config (or reset to defaults if no new config is passed)"?

This would require also changing the `--addon-config` description to something like:
"A path to a configuration file of add-ons. If an add-on is already installed, this new config gets merged with the existing one (unless --addon-overwrite is used)"